routes/web/project/user.rb

```diff
@@ -12,7 +12,8 @@ class CloverWeb
 
     r.post true do
       email = r.params["email"]
-      user = Account[email: email]
+      # Don't invite deleted accounts
+      user = Account.exclude(status_id: 3).first(email: email)
```

Comment:
`Account.exclude(status_id: 3)[email: email]` ?

Reply:
I didn't realize that we could use `[]` with a dataset. I thought it could only be used directly with the class. Thanks for the new insight; I made the change.